laravel/app/Console/Commands/UpdateAnesthesiologistsData.php

```diff
@@ -43,107 +43,99 @@ public function handle()
-                    break;
-                case "99ccff":
-                    $sRet = "CRNA"; //General
-                    break;
-                case "9966cc":
-                    $sRet = "CA-1";
-                    break;
-                case "cc99cc":
-                    $sRet = "CA-1";
-                    break;
-                case "ccccff":
-                    $sRet = "Attending"; //PartTime
-                    break;
-                case "ccffff":
-                    $sRet = "CRNA"; //PartTime
-                    break;
-                case "f5faf5":
-                    if (strpos($oStaffMember["Abbrev"], "SRNA") !== false) {
-                        $sRet = "SRNA";
-                    } else {
-                        $sRet = "Attending"; //General
-                    }
-                    break;
-                case "ff0000":
-                    $sRet = "CRNA"; //Cardiac
-                    break;
-                case "ff99cc":
-                    $sRet = "CA-2";
-                case "ff9999":
-                    if ($oStaffMember["ExtCallSysId"] == "PGY") {
-                        $sRet = "CA-2";
-                    } else {
-                        $sRet = "Attending"; //Cardiac
-                    }
-                    break;
-                case "ffcc99":
-                    if (strpos($oStaffMember["Abbrev"], "Med Student") !== false) {
-                        $sRet = "Student";
-                    } else if (strpos($oStaffMember["Abbrev"], "Outside") !== false) {
-                        $sRet = "Student";
-                    } else {
-                        $sRet = "CA-3";
-                    }
-                    break;
-                case "ffcccc":
-                    $sRet = "CA-1";
-                    break;
-                case "ffccff":
-                    $sRet = "Fellow";
-                    break;
-                case "ffff99":
-                    $sRet = "Attending"; //East
-                    break;
-                case "ffffff":
-                    $sRet = "Attending";
-                    break;
-                default:
-                    $sRet = "";
-                    break;
-            }
+        //get scheduling date in format: "yyyy-mm-ddT00:00:00"
+        $date = substr(date("c", strtotime('+2 day')), 0, -14) . "00:00:00";
+        if(date("l", strtotime('today')) == 'Thursday' || date("l", strtotime('today')) == 'Friday'){
+            $date = substr(date("c", strtotime('+4 day')), 0, -14) . "00:00:00";
+        } elseif (date("l", strtotime('today'))=='Saturday') {
+            $date = substr(date("c", strtotime('+3 day')), 0, -14) . "00:00:00";
+        }
+        // filter for available attending for scheduling day
+        foreach ($json_data as $staffMember) {
+            if(strval($staffMember["Date"]) == $date){
+                $taskAbbrev = strval($staffMember["TaskAbbrev"]);
+                $label = "";
+                switch($taskAbbrev){
```

Comment:
could we replace the long switch with something like:
```
if ($attendingTaskAbbrevs.contains($taskAbbrev)) {
    // add to Anest table
}
```
where `$attendingTaskAbbrevs` is a [Set ](https://www.php.net/manual/en/class.ds-set.php) of strings?